move2kube/exercise-3-customizations/exercise-3a-overriding-built-in-transformers/customizations/custom-dockerfile-change-built-in-behavior/nodejs/templates/Dockerfile

```diff
@@ -0,0 +1,22 @@
+#   Copyright IBM Corporation 2020
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+FROM quay.io/konveyor/nodejs-12
```

Comment:
Please check the nodejs version in the Dockerfile generated by Move2kube without using the custom transformer, and update the nodejs version in the base image in the custom Dockerfile template if required.

Reply:
```
FROM quay.io/konveyor/nodejs-12
COPY . .
RUN npm install
RUN npm run build
EXPOSE 8080
CMD sh start-nodejs.sh
```

Its the same 😄